.changeset/many-carpets-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
```

Comment:
It's a bit petty, but someone could have saved a direct link to this docs page. Removing the docs page is therefore breaking the link.

```suggestion
'@swisspost/design-system-documentation': major
```